internal/pkg/manager/spod/spod_controller.go

```diff
@@ -101,7 +101,7 @@ func (r *ReconcileSPOd) Healthz(*http.Request) error {
 // Operand:
 // +kubebuilder:rbac:groups="",resources=services,verbs=get;list;watch;create;update;patch
 // +kubebuilder:rbac:groups=apps,resources=deployments;daemonsets,verbs=get;list;watch;create;update;patch
-// +kubebuilder:rbac:groups=apps,resources=deployments;daemonsets/finalizers,verbs=delete;get;update;patch
+// +kubebuilder:rbac:groups=apps,resources=deployments/finalizers;daemonsets/finalizers,verbs=delete;get;update;patch
```

Comment:
I think this was a bug in the first place, I don't think we need to delete deploys at the moment and the other verbs were already pretty much in the line above